ansible/ansible-version.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: all
```

Comment:
I think that this will not test the localhost which is the only that this roles wants to test?